README.md

```diff
@@ -1,30 +1,30 @@
-# ad440-winter2022-thursday-repo
```

Comment:
I have made necessary changes that eknigge and CalebMcOlin made as to explanations to review in regards to README.md file and how to implement the Yaml file in Git Bash deployment.
I appreciate your review. 1jc

I have resolved and made the necessary corrections to delete 2 commits per requested review plus additional explanations about README and deployment-testing of Yaml file in Git Bash or CLI. I still need two approvals please
Thank you  1jc

Reply:
Hello @dagmawidemissie17  I don-t really need this file since I have printed everything out in the open in the #76 comments to be read by everyone.  I will resolve this issue by deleting this committed file  Thanks 1jc

Reply:
@dagmawidemissie17 resolve conflict deleted the README.md file
Appreciate your help   1jc

Reply:
@dagmawidemissie17 @CalebMcOlin  resolved conflict deleted commit file for README.md

Thanks,  appreciate your input     1jc